application/src/main/java/uk/nhs/digital/uec/api/filter/AccessTokenFilter.java

```diff
@@ -30,6 +31,7 @@
 
 @Component
 @Slf4j
+@Profile("!local")
```

Comment:
Because we may want to wire in the mock authentication for more than one profile, shall we give this a more specific profile name, such as: 'mock-auth', we can then spin up our API to use this mock auth implementation in a range of environments, for example: SPRING_ACTIVE_PROFILES = local,mock-auth; or SPRING_ACTIVE_PROFILES = dev,mock-auth